functions/src/default.ts

```diff
@@ -98,7 +98,14 @@ If you can't get the PR to a green state due to flakes or broken master, please
     // arrays of labels that determine if an issue has been triaged by the caretaker
     l1TriageLabels: [["comp: *"]],
     // arrays of labels that determine if an issue has been fully triaged
-    l2TriageLabels: [["type: bug/fix", "severity*", "freq*", "comp: *"], ["type: feature", "comp: *"], ["type: refactor", "comp: *"], ["type: RFC / Discussion / question", "comp: *"]]
+    l2TriageLabels: [
```

Comment:
Since these are just the default values, why don't we just modify them in the `angular/angular` repository config file?

The `angular/angular` config file already actually sets its own it appears ([link](https://github.com/angular/angular/blob/master/.github/angular-robot.yml#L142))

Reply:
Ah! Yes, that's definitely better; I wasn't aware of that. I'll make the change there